TestHelper/CoreLocation.swift

```diff
@@ -136,14 +136,18 @@ extension Array where Element == CLLocation {
     
     // Shifts the [CLLocation]’s first location to now and offsets the remaining locations by one second after the prior.
     public func shiftedToPresent() -> [CLLocation] {
-        let now = Date()
+        return shiftedTo(Date())
+    }
+    
+    // Shifts the [CLLocation]’s first location to the given timestamp and offsets the  remaining locations by one second after the prior.
+    public func shiftedTo(_ timestamp: Date) -> [CLLocation] {
```

Comment:
Nit:

```swift
public func shifted(to timestamp: Date) -> [CLLocation] {
```

Reply:
I don't disagree with the improved swiftiness of this suggestion, but wouldn't that necessitate a more objc-friendly variant?

Reply:
Renamed to `shifted(to:)`. This is in the TestHelper which isn't being used in Obj-C currently.